liblocation/src/main/java/com/mapbox/android/core/location/GoogleLocationEngine.java

```diff
@@ -102,17 +109,28 @@ public void onConnectionSuspended(int cause) {
   public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
   }
 
+  @SuppressLint("MissingPermission")
```

Comment:
instead of suppressing Lint can we add the required permissions annotations instead?